zipkin/src/main/java/zipkin/internal/InetAddresses.java

```diff
@@ -0,0 +1,191 @@
+/**
+ * Copyright 2015-2017 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package zipkin.internal;
+
+import java.net.InetAddress;
+import java.nio.ByteBuffer;
+
+/** Utilities for working with IP Addresses. */
+public final class InetAddresses {
```

Comment:
note an identical copy of this patched class was in Brave for quite a while now.